docs/content/get-started/install.md

```diff
@@ -59,7 +59,7 @@ The following sections describe how to install the CLI on Linux, macOS, and Wind
 1. Test the installation.
 
     ```bash
-    nginx-meshctl version
+    nginx-meshctl
```

Comment:
Can actually use our `nginx-meshctl status` command. Just running the binary spits out the usage statement.

Reply:
Actually tried that first, but if service mesh isn't installed...
![image](https://user-images.githubusercontent.com/7340923/225693272-b241499b-6f3c-4285-a75a-bd4172a94fe8.png)